packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/JavaScriptChannel.java

```diff
@@ -0,0 +1,32 @@
+package io.flutter.plugins.webviewflutter;
+
+import android.webkit.JavascriptInterface;
+import io.flutter.plugin.common.MethodChannel;
+import java.util.HashMap;
+
+/**
+ * Added as a JavaScript interface to the WebView for any JavaScript channel that the Dart code sets
+ * up.
+ *
+ * <p>Exposes a single method named `postMessage` to JavaScript, which sends a message over a method
+ * channel to the Dart code.
+ */
+class JavaScriptChannel {
+  private final MethodChannel methodChannel;
+  private final String javaScriptChannelName;
+
+  JavaScriptChannel(MethodChannel methodChannel, String javaScriptChannelName) {
```

Comment:
nit: Can you also add docs explaining these params? I think it's just tricky enough that it's worth saying what they do here.

Reply:
done